internal/compiler/builtins.slint

```diff
@@ -122,6 +122,8 @@ export TouchArea := _ {
 export struct KeyboardModifiers := {
     //-name:slint::private_api::KeyboardModifiers
     alt: bool,
+    alt_gr: bool,
+    caps_lock: bool,
```

Comment:
I don't know if we need to add this in this struct. They are not meant to be part of shortcut, are they? They are already interpreted by the OS to change the "text"